src/shogun/features/DotFeatures.cpp

```diff
@@ -358,19 +365,19 @@ void CDotFeatures::benchmark_dense_dot_range(int32_t repeats)
 			dense_dot_range(out, 0, num, alphas, w, d, 23);
 
 #ifdef DEBUG_DOTFEATURES
-    CMath::display_vector(out, 40, "dense_dot_range");
```

Comment:
This is not improving this is breaking history of the code... Which makes traceability of a given change quite cumbersome. 

> On Mar 12, 2016, at 15:50, Kunal Arora <notifications@github.com> wrote:
> 
> In src/shogun/features/DotFeatures.cpp:
> 
> > @@ -358,19 +365,19 @@ void CDotFeatures::benchmark_dense_dot_range(int32_t repeats)
> >  			dense_dot_range(out, 0, num, alphas, w, d, 23);
> >  
> >  #ifdef DEBUG_DOTFEATURES
> > -    CMath::display_vector(out, 40, "dense_dot_range");
> @vigsterkr any particular reason why we should not try to improve existing code, even if it is just indentation? Am I missing something here?
> 
> —
> Reply to this email directly or view it on GitHub.
>